src/main/ui/app/component/AddOnList.jsx

```diff
@@ -1,21 +1,43 @@
 import {Component} from "react";
 import AddOn from "./AddOn";
+import { ListGroup , Table} from 'react-bootstrap';
+
 
 export default class AddOnList extends Component {
 
     render() {
         if (!this.props.addons) {
-            return <div>Loading...</div>
+            return (
+             <section>
+      <p class="lead textdec">Loading</p>
+      </section>
+      )
         }
         else if (this.props.addons.length === 0) {
-            return <div>No results</div>
+            return (
+            <section>
+      <h1 class="lead textdec">No Results</h1>
+      </section>
+
+            )
         }
         else {
             return (
-                    <div>
+                   <div>
+		   <Table className="maintable">
+    			<thead>
+      			<tr>
+			<th><i className='fa fa-2x fa-globe' aria-hidden="true"></i></th>
```

Comment:
Git rid of the globe icon here. Better for it to be blank, since we don't want to draw the user's eye here.